examples/tutorials/run-scalable-flow-with-pipeline/pipeline.ipynb

```diff
@@ -0,0 +1,613 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Why use Azure machine learning(ML) pipelines to run your flows on the cloud?\n",
+    "In real-world scenarios, flows serve various purposes. For example, consider a flow designed to evaluate the relevance score for a communication session between humans and agents. Suppose you want to trigger this flow every night to assess today’s performance and avoid peak hours for LLM (Language Model) endpoints. In this common scenario, people often encounter the following needs:\n",
```

Comment:
make our reference doc point to this example?

Reply:
Do you mean the doc at here: https://microsoft.github.io/promptflow/index.html
If so, I'll plan the update later

Reply:
And a question for doc & repo, which one would be the first entrance from customer?